### ISSUE TITLE
params provided as list with parameter of single value

### ISSUE THREAD

**Issue by BenSchZA** [cadCAD-org/cadCAD#175]
When setting the initial parameters of the system to a single value, i.e. no parameter sweep, the params are provided as a list to the state update and policy functions rather than a dictionary. This isn't expected correct?

The following error is thrown `TypeError: list indices must be integers or slices, not str` which is simply solved by accessing the specific parameter after selecting the first list index:

```python
m = {
    'a': [2]
}

...

def policy_function_1(_params, substep, sH, s, **kwargs):
    a = _params[0]['a']
    return {'a': a}
```

**Comment by BenSchZA** [comment]
I'm sure it's a simple fix, and don't mind looking into it either.

**Comment by BenSchZA** [comment]
The issue actually seems to be quite unpredictable. I'm getting the same error now with a parameter sweep. I made a few minor changes to the model, but still using a minimum viable model.

**Comment by JEJodesty** [comment]
@BenSchZA 
I'll handle it this time because its for a feature extension.
can you included a link to the system model?

**Comment by BenSchZA** [comment]
Cool, I don't have a repo for it, but will upload a recreation of the error for you.

**Comment by JEJodesty** [comment]
@BenSchZA 
that would be greatly appreciated

**Comment by JEJodesty** [comment]
This issue exists in `staging`

**Comment by BenSchZA** [comment]
@JEJodesty https://gist.github.com/BenSchZA/9a67a30e64ad7a7876dc3b72d1e303d5 - note that I did a `pip install https://github.com/cadCAD-org/cadCAD/archive/master.zip --force-reinstall` because of the issue with PyPi version.

**Comment by aclarkData** [comment]
Same here. The issue did not go away in 0.4.20

**Comment by JEJodesty** [comment]
@aclarkData @BenSchZA 
i wasn't aiming to solve this with v0.4.20
I will hotfix this with v0.4.21

**Comment by JEJodesty** [comment]
@aclarkData 
Can you provide a repo for me to solve this? (Offline)

**Comment by aclarkData** [comment]
@JEJodesty https://github.com/BlockScience/Aragon_Conviction_Voting/blob/params/models/v3/Aragon_Conviction_Voting_Model.ipynb

**Comment by JEJodesty** [comment]
@matttyb80 @BenSchZA @aclarkData
try it now
1.
reinstall cadCAD
Option B: Build From Source
https://github.com/cadCAD-org/cadCAD/tree/hot_fix

**Comment by aclarkData** [comment]
Using the hot_fix, it's working

**Comment by JEJodesty** [comment]
Hot fix #183
